### PR TITLE
Add travis sqf and config checking

### PR DESCRIPTION
**Pull request by Glowbal**
**When merged this pull request will:**
- Replace some tabs in sqf files by 4 spaces
- Add the SQF and config checking tools from ACE
- Add travis configuration for checking sqf and config files (will enable once this is merged)

### CODE REVIEW

File: tools/sqf_validator.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+import fnmatch
+import os
+import re
+import ntpath
+import sys
+import argparse
+
+def validKeyWordAfterCode(content, index):
+    keyWords = ["for", "do", "count", "each", "forEach", "else", "and", "not", "isEqualTo", "in", "call", "spawn", "execVM", "catch", "param", "select", "apply"];
```
Comment (Dorbedo) [inline tools/sqf_validator.py]: isn't "or" missing?

**Comment by Killswitch00** [comment]
Not too keen on having what should be two different changesets in one PR (whitespace cleanup and files related to the actual PR description).

**Comment by Glowbal** [comment]
White space clean up is for the validation scripts to pass.

**Comment by Glowbal** [comment]
I will enable Travis for CBA.

**Pull request merged**